cinema/views.py

```diff
@@ -34,6 +36,33 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
This method is duplicated in other viewset. Move it to another place and reuse in both viewsets